_partners/sematext.md

```diff
@@ -0,0 +1,81 @@
+---
+name: Sematext
+# name_long is used on the partner details page.
+name_long: 'Sematext Group'
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/sematext.png'
```

Comment:
I think you meant `octi-square-logo.png`

Reply:
Ah, right, thanks.  Re-added with the correct name now.